nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -48,6 +48,8 @@ let
     # User and group
     User = cfg.user;
     Group = cfg.group;
+    # Working directory
```

Comment:
We don't need to repeat the actual code in a comment. Similar existing comments should be dropped, too.